how_to_add_new_hotspot.md

```diff
@@ -0,0 +1,142 @@
+
+### Example Variant
+Here, you can find an example for such variant:
+
+```
+'NEBHNT-IN1': {
+    'FRIENDLY': 'Nebra Indoor Hotspot Gen 1',
+    'APPNAME': 'Indoor',
+    'SPIBUS': 'spidev1.2',
+    'KEY_STORAGE_BUS': '/dev/i2c-1',
+    'RESET': 38,
+    'MAC': 'eth0',
+    'STATUS': 25,
+    'BUTTON': 26,
+    'ECCOB': True,
+    'TYPE': 'Full',
+    'CELLULAR': False,
+    'FCC_IDS': ['2AZDM-HNTIN'],
+    'CONTAINS_FCC_IDS': ['2AHRD-EPN8531', '2AB8JCSR40', '2ARPP-GL5712UX'],
+    'IC_IDS': ['27187-HNTIN'],
+    'CONTAINS_IC_IDS': []
+    },
+```
+
+### Variant Parameters
+
+**Important Note:** GPIO index numbers used for parameters like `BUTTON` or `STATUS` are getting their values from CPU GPIO peripheral. So the developer should not mix them with CPU's or any connectors (like HAT connector) pin numbers. The corresponding GPIO index number of a particular physical pin could be found on the hardware design document of the new hotspot.
+
+#### FRIENDLY
+It is the human readable and explanatory name of the device.
+
+#### APPNAME
+Explains usage detail
+
+#### SPIBUS
+The LoRa concentrator hardware is designed to work on a SPI (Serial Peripheral Interface) Bus. This has to be defined as found in Linux IO device, which should be found in `/dev` path.
+
+#### KEY_STORAGE_BUS
+All hotspots has to have a valid cryptography integrated circuit (IC), which is called ECC. This IC works on I2C bus and this parameter is defining the Linux IO device path for this particular I2C bus. Unlike SPIBUS, this path has to be absolute.
+
+#### RESET
+This is the GPIO index of the digital output, which is connected to the RF Concentrator chip (e.g. SX1301, SX1302 ...) reset.
+
+For a correct RF Concentrator opearation, the concentrator chip has to be reset between configuration changes. It is reset at the start of the `packet_forwarder` container. This is the primary source of the problem if a concentrator is not functioning as expected.
+
+#### MAC
+This parameter defines which network interface is used for MAC (Media Access Control) code. It has to be found in the devices network interfaces.
+
+#### STATUS
+This is the hotspots status LED's output GPIO index, which is usually a single green LED on most SBC's.
+
+#### BUTTON
+This parameter defines the GPIO input for Bluetooth advertisement start.
+
+Hotspots needed to be accessed via Bluetooth sometimes. Especially at the beginning, onboarding time. But keeping the required Bluetooth interface on all the time imposes two problems. First, it is a security thread and leaks some information to trace the device. Second it also effects power consumption negatively as the device is expected to work 7/24.
+
+This input triggers the Bluetooth advertisement process and then the user can access it via Bluetooth interface.
+
+#### ECCOB
+Indicates whether the ECC is onboard or not. It is a bool parameter.
+
+#### TYPE
+Defines Helium Miner type (e.g. Full, Lite etc.)
```

Comment:
I think we should call it Light not lite personally (and i believe helium call it Light)

Reply:
Yea, found that out earlier but forgot to change.